### PR TITLE
feat(lib): implement sorting of config diff object

### PR DESCRIPTION
**Pull request by ta2edchimp**
Relates to #50

### CODE REVIEW

**Comment by ta2edchimp** [comment]
Will rebase and push again.

File: src/lib/sort-rules.js [hunk 1]
```diff
@@ -1,9 +1,29 @@
...
-    return a > b ? 1 : -1
-  })
+  return Array.isArray(rules) ? getSortedRulesArray(rules) : transformIntoSortedRulesArray(rules)
+}
+
+function getSortedRulesArray(rules) {
+  return rules.sort(sortAlphabetically)
+}
+
+function transformIntoSortedRulesArray(allRulesArray) {
```
Comment (sarbbottam) [inline src/lib/sort-rules.js]: I guess the name `allRulesArray` is not semantic, its and `object`, may be you want to keep it `rules`

File: src/lib/sort-rules.js [hunk 2]
```diff
@@ -1,9 +1,29 @@
...
+  var sortedRules = []
+
+  Object.keys(allRulesArray)
+    .sort(sortAlphabetically)
+    .forEach(function mapRules(ruleName) {
+      var singleRuleArray = {}
```
Comment (sarbbottam) [inline src/lib/sort-rules.js]: same here.

Comment (ta2edchimp) [inline src/lib/sort-rules.js]: And this is the kind of mistakes someone makes when getting distracted while coding :/

**Comment by sarbbottam** [comment]
👍 after rebase and typo fix!

**Comment by sarbbottam** [comment]
My bad, clicked the wrong button by mistake.

**Comment by ta2edchimp** [comment]
Merged, as coordinated with @sarbbottam

**Pull request merged**